### PR TITLE
Rework eth_wallet signal

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary
Take the previous balance (time_window time ago) instead of the balance
when the signal was previously triggered

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/clickhouse/metric/available_v2_metrics.json [hunk 1]
```diff
@@ -276,6 +276,7 @@
   },
   {
     "metric": "stack_realized_cap_usd_20y",
+    "alias": "realized_value_usd",
```
Comment (tspenov) [inline lib/sanbase/clickhouse/metric/available_v2_metrics.json]: that should be in another PR :?

Comment (IvanIvanoff) [reply lib/sanbase/clickhouse/metric/available_v2_metrics.json]: yes
